File: innn/intersect-android/app/src/main/java/com/tmmt/innersect/ui/adapter/FooterDelegateAdapter.java
package com.tmmt.innersect.ui.adapter;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.tmmt.innersect.mvp.model.CommonAdapterItem;
import com.tmmt.innersect.ui.adapter.viewholder.CommonViewHolder;


/**
 * Created by flame on 2017/7/24.
 */

public class FooterDelegateAdapter implements ViewTypeDelegateAdapter {


    public FooterDelegateAdapter(int layoutId){
        mLayoutId=layoutId;
    }

    private int mLayoutId;

    @Override
    public void onBindViewHolder(CommonViewHolder holder, CommonAdapterItem item) {}

    @Override
    public CommonViewHolder onCreateViewHolder(ViewGroup parent) {
        View view= LayoutInflater.from(parent.getContext()).inflate(mLayoutId,parent,false);
        return new CommonViewHolder(view);
    }

}

File: app/src/main/java/com/stesso/android/lib/FooterDelegateAdapter.kt
package com.stesso.android.lib

import android.view.ViewGroup
import android.widget.FrameLayout
import com.stesso.android.utils.dip2px

class FooterDelegateAdapter : ViewTypeDelegateAdapter {

    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): CommonViewHolder {

        val padding = dip2px(40)
        val view =FrameLayout(parent.context)
        view.setPadding(padding,padding,padding,padding)
        return CommonViewHolder(view)
    }

    override fun onBindViewHolder(holder: CommonViewHolder, position: Int, data :Any?) {

    }


}